### PR TITLE
update DRF and Haystack requirements

### PR DESCRIPTION
**Pull request by staubina**

### CODE REVIEW

File: course_discovery/apps/api/tests/test_serializers.py [hunk 1]
```diff
@@ -64,10 +64,8 @@ def make_request():
     return request
 
 
-def serialize_datetime_without_timezone(d):
-    # TODO: Remove this function, and replace usage of it with serialize_datetime, after
-    # https://github.com/encode/django-rest-framework/issues/3732 is released.
-    return d.strftime('%Y-%m-%dT%H:%M:%S') if d else None
+def serialize_datetime(d):
+    return d.strftime('%Y-%m-%dT%H:%M:%SZ') if d else None
```
Comment (mikix) [inline course_discovery/apps/api/tests/test_serializers.py]: This already exists in `from course_discovery.apps.core.utils import serialize_datetime`

**Review by mikix — APPROVED** [review]
Nice!

File: course_discovery/apps/api/tests/test_serializers.py (hunk 1 above)
Comment (staubina) [reply course_discovery/apps/api/tests/test_serializers.py]: Thank you! I was looking for it and did not find it.

File: course_discovery/apps/api/tests/test_serializers.py [hunk 2]
```diff
@@ -50,6 +51,7 @@
     VideoFactory
 )
 from course_discovery.apps.course_metadata.utils import get_course_run_estimated_hours
+
```
Comment (mikix) [inline course_discovery/apps/api/tests/test_serializers.py]: ?

**Review by mikix — APPROVED** [review]

**Pull request merged**